include/nuttx/rptun/rptun.h

```diff
@@ -30,7 +30,9 @@
 #ifdef CONFIG_RPTUN
 
 #include <nuttx/fs/ioctl.h>
-#include <openamp/open_amp.h>
+#include <nuttx/rptun/rptun_ping.h>
```

Comment:
there is no such file in upstream

Reply:
I see that this file is added in https://github.com/apache/nuttx/pull/11537 but this line should not be in this PR

Reply:
Yes, this PR should be submited after #11537.